plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -279,15 +279,17 @@ struct Database {
     }
 
     /// Returns: the stored scores in ascending order by their `time`.
-    FileScore[] getMutationFileScoreHistory() @trusted {
-        auto app = appender!(FileScore[])();
-        auto stmt = db.prepare(format!"SELECT AVG(score), file_path FROM (SELECT * FROM %s WHERE time_stamp > date('now', '-7 day')) GROUP BY file_path"(
-                mutationFileScoreHistoryTable));
-        foreach (a; stmt.get.execute) {
-            app.put(FileScore(SysTime(0),
-                    typeof(FileScore.score)(a.peek!double(0)), Path(a.peek!string(1))));
+    FileScore[][Path] getMutationFileScoreHistory() @trusted {
+        FileScore[][Path] scoreDict;
+
+        auto stmt = db.prepare(
+                "SELECT * FROM " ~ mutationFileScoreHistoryTable ~ " ORDER BY time_stamp ASC");
```

Comment:
use the column names instead of "*". it makes it a lot easier in the future when adding/removing columns.